models/classes/accessControl/ContextInterface.php

```diff
@@ -22,8 +22,9 @@
 
 namespace oat\tao\model\accessControl;
 
-use oat\oatbox\user\User;
-
+/**
+ * @deprecated Use \oat\tao\model\Context\ContextInterface
+ */
 interface ContextInterface
```

Comment:
You can extends the proper interface and remove the methods from here, so it is easier/safer to migrate to eliminate deprecated implementations.

```suggestion
interface ContextInterface extends \oat\tao\model\Context\ContextInterface
```